custom/icds/views/custom_sms_report.py

```diff
@@ -40,6 +40,12 @@ def page_context(self):
         }
 
     def post(self, request, *args, **kwargs):
+        reports_in_progress = self.report_tracker.active_reports
+        if len(reports_in_progress) >= 3:
+            messages.warning(self.request,
```

Comment:
nit: should be message.error since you are blocking the action and not just warning about it?